index.js

```diff
@@ -17,6 +18,8 @@ once('__VUE_EXTEND_DROPPOINT_VPB__', () => {
   Vue.use(VueProgressBar)
 })
 
+Vue.use(VueLazyload)
```

Comment:
add same options that are used in core `Vue.use(VueLazyload, { attempt: 2, preLoad: 1.5 })`